libraries/apollo-mockserver/src/jsMain/kotlin/com/apollographql/apollo3/mockserver/SocketServer.js.kt

```diff
@@ -0,0 +1,95 @@
+package com.apollographql.apollo3.mockserver
+
+import Buffer
+import kotlinx.coroutines.channels.Channel
+import net.AddressInfo
+import net.createServer
+import okio.IOException
+import org.khronos.webgl.Int8Array
+import org.khronos.webgl.Uint8Array
+import org.khronos.webgl.set
```

Comment:
Probably it's better to ditch `kotlin-nodejs` (unmaintained) and instead rely on `kotlin-wrappers`.  
That means you'll be able to use `js.typedarrays.*` externals instead of WebGL stuff.

Reply:
Also see https://github.com/JetBrains/kotlin-wrappers/pull/2133 in case you go with `kotlin-wrappers`.

Reply:
TIL about kotlin-wrappers, this is much better indeed, thanks 🙏  Also allows us to remove the jcenter dependency 🎉